base/ltmeta.dtx

```diff
@@ -119,8 +119,12 @@
 %<latexrelease>                 {Document Metadata handling}
 %    \end{macrocode}
 %
+% \changes{v1.0d}{2025/03/01}{Provide T F variants of
+%                             \cs{IfDocumentMetadataTF} (gh/1522)}
```

Comment:
This all seems to need more explanation (or maybe a reference to somewhere?).

Reply:
added some explanation